### PR TITLE
Milestone 4 Calculator project

### PR DESCRIPTION
**Pull request by imhilla**
Hello TSE,
Please review this forth milestone for the React calculator project.
I added the logic to handle calculations and operations. Any feedback will be highly appreciated.

Thanks
Hillary

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import Operate from './operate';
+
+const Calculate = (() => {
+  const calculate = (dataObject, buttonName) => {
+    // eslint-disable-next-line
+    var { total, next, operation } = dataObject;
+    if (buttonName === operation) {
+      Operate.operate(total, next, operation);
+    }
```
Comment (Torres-ssf) [inline src/logic/calculate.js]: - [ ] We need to evaluate more cases here. What should happen if the user press `AC`, `.` or `+/-`? We also need to address these cases.

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      total = one - two;
+    } else if (operation === '+') {
+      total = one + two;
+    } else if (operation === 'X') {
+      total = one * two;
+    } else if (operation === '÷') {
+      total = one / two;
+    } else if (operation === '%') {
+      total = (one / 100) * two;
+    }
```
Comment (Torres-ssf) [inline src/logic/operate.js:19]: - [ ] Most likely the modifier operation is going to throw an error if number two was not typed yet.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi there @imhilla :hand: 
Awesome job on this milestone so far :clap: 
But we need to make some changes before moving on

- [ ] Check my comments bellow

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required :heavy_multiplication_x: 
Great job on this project :clap: 
Let's make a few changes before we proceed to the next milestone.

**REQUIRED**
* The button name "+/-" should multiply total and next by -1 to make it negative or positive. Currently, an error is thrown upon an attempt to use this functionality.
* Let's add some information on how to get the app running locally to our README file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**After making the changes, submit another request for review.**

**Happy Hacking :computer: 
Simon Wathigo.**

**Review by mosaaleb — APPROVED** [review]
### Well done @imhilla :clap:

Looks like you have handled all requirements from the previous reviews.
You can merge :+1:

##### Muhammad Ebeid

**Pull request merged**